logstash_async/memory_cache.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# This software may be modified and distributed under the terms
+# of the MIT license.  See the LICENSE file for details.
+
+import uuid
+from datetime import datetime, timedelta
+
+
+class MemoryCache(object):
```

Comment:
Hmm... 
maybe good idea would be create abstract base class?
what do you think?